src/platform/site-wide/mega-menu/README.md

```diff
@@ -0,0 +1,17 @@
+# Mega menu configuration
```

Comment:
It might be nice for `Configuration` to be moved into a subsection and then maybe a summary of what the `Mega menu` is at the top, because what the Mega Menu is might be unclear to someone that hasn't heard it before